R/pkg/R/generics.R

```diff
@@ -581,6 +579,10 @@ setGeneric("unionAll", function(x, y) { standardGeneric("unionAll") })
 #' @export
 setGeneric("where", function(x, condition) { standardGeneric("where") })
 
+#' @rdname with
+#' @export
+setGeneric("with")
```

Comment:
Fixed this and also re-ordered generics declaration for attach and as.data.frame.